ddtrace/llmobs/_llmobs.py

```diff
@@ -560,7 +561,7 @@ def _tag_embedding_io(cls, span, input_documents=None, output_text=None):
                 span.set_tag_str(OUTPUT_VALUE, output_text)
             else:
                 try:
-                    span.set_tag_str(OUTPUT_VALUE, json.dumps(output_text))
+                    span.set_tag_str(OUTPUT_VALUE, json.dumps(output_text, default=_unserializable_default_repr))
                 except TypeError:
```

Comment:
should we expand this try/catch to be more generic (maybe just to Exception)? I don't think we should be expecting TypeError's anymore because of the `default` function, and `_unserializable_default_repr` doesn't throw a `TypeError`.

Reply:
`_unserializable_default_repr()` does throw a `TypeError` if `repr(obj)` is not a string object. The exceptions should still stay there, as we need to catch those cases.